### ISSUE TITLE
Update veda-docs for production release preparation

### ISSUE THREAD

**Issue by botanical** [NASA-IMPACT/veda-docs#145]
### Context
As we prepare to release production, we need to update our veda-docs to link to the correct APIs and ensure that the notebooks are also functioning. 

### Acceptance Criteria

- [x] Update api links on [landing page](https://nasa-impact.github.io/veda-docs/services/apis.html#production-stable-coming-soon)
  - [x] Veda Production
    - [x] STAC API (production catalog) -> https://openveda.cloud/api/stac
    - [x] Raster API -> https://openveda.cloud/api/raster/docs, 
    - [x] Browser -> https://openveda.cloud/
  - [x] Staging STAC API (staging catalog for vetting new data before public release) https://staging-stac.delta-backend.com/ --> https://staging.openveda.cloud/api/stac
  - [x] Staging Raster API https://staging-raster.delta-backend.com/docs --> https://staging.openveda.cloud/api/raster/docs
  - [x] Staging Browser http://veda-staging-stac-browser.s3-website-us-west-2.amazonaws.com/ --> https://staging.openveda.cloud/
- [x] Update api links in notebooks

**Comment by botanical** [comment]
Updates:
- Waiting on getting veda-config and veda-config-eic migrated
- Additionally, maybe good to get veda-docs updated ? But not entirely a blocker

**Comment by anayeaye** [comment]
[Sheet tracking progress on notebook updates](https://docs.google.com/spreadsheets/d/1i-H4Q8a5PCS4uZsDEpa2ZA-MVUP41zd63fnS_QCiGNM/edit?gid=0#gid=0)

**Comment by smohiudd** [comment]
The only notebook still outstanding is `./notebooks/tutorials/stac_ipyleaflet.ipynb`. Which requires an update to pangeo notebook:  https://github.com/NASA-IMPACT/pangeo-notebook-veda-image/pull/27. I will mark this ticket a done and we track in the pangeo ticket and a new issue if necessary.
